config/remotesigner.go

```diff
@@ -10,11 +10,13 @@ const (
 type RemoteSignerCfg struct {
 	URL     string        `long:"url" description:"URL of the remote signer"`
 	Timeout time.Duration `long:"timeout" description:"client when making requests to the remote signer"`
+	HMACKey string        `long:"hmackey" description:"HMAC key for authenticating requests to the remote signer - can be a direct key or reference to a cloud secret manager. Leave empty to disable HMAC authentication."`
```

Comment:
hmm when this is configured with `reference to a cloud secret manager` , it will contact configured secrets manager automatically ?

Reply:
Yes that's the plan to have it automatically recognize the format of secret depending on the provider (ARN for AWS for example)